assignment.js

```diff
@@ -112,7 +124,46 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
-
+let inputA = 'scissors'
+let inputB = 'paper'
+
+inputA = inputA.toLowerCase();
+inputB = inputB.toLowerCase();
+
+function inputCheck() {
+  if((inputA === 'rock' || inputA === 'paper' || inputA === 'scissors') && (inputB === 'rock' || inputB === 'paper' || inputB === 'scissors')) {
+    return inputA, inputB
+  } else {
+    console.log('Error. Enter rock, paper, or scissors.')
+  }
+}
+
+inputCheck()
+
+if(inputA === inputB) {
```

Comment:
@studiosok  nice approach covering all of the possible outcomes. 

The approach that you're using involves nested `if` statements and while it works, you will want to avoid nesting if possible. The reason is that nesting can add complexity to your code which makes things more challenging to debug. In this case, the code was still fairly easy to follow.

Reply:
Thanks! Really appreciate the alternative solutions (other problems) and the why behind using nesting sparingly (this one).